content/en/docs/openshift/install-kubeflow.md

```diff
@@ -95,7 +95,7 @@ Use the following steps to install Kubeflow 1.0 on OpenShift 4.x.
     ...
     ```
 
```

Comment:
Lines 2-3: 

- To be specific and to help the search engines find the guides - mention Openshift in the title and the description (similar to the GCP, and other guides)
- "Install Kubeflow" may be too generic

```diff
- title = "Install Kubeflow"
- description = "Instructions for deploying Kubeflow from the command line"
+ title = "Install Kubeflow on OpenShift"
+ description = "Instructions for deploying Kubeflow on an OpenShift cluster from the command line"
```

Reply:
Done :heavy_check_mark: